{"name": "test_mul[negative]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "self = <HogwartsLG6.taskwork.stage4_live_pytest.test_calc_new.TestCalc object at 0x0000024858F19640>\nget_calc1 = <HogwartsLG6.taskwork.stage4_live_pytest.common.calc.Calculator object at 0x0000024858E97520>\nget_mul_datas = [-1, -1, 2]\n\n    @allure.story(\"测试乘法\")\n    def test_mul(self, get_calc1, get_mul_datas):\n        with allure.step(\"计算两个数相乘积\"):\n            result = get_calc1.mul(get_mul_datas[0], get_mul_datas[1])\n        if isinstance(result, float):\n            result = round(result, 2)\n>       assert result == get_mul_datas[2]\nE       assert 1 == 2\n\ntest_calc_new.py:47: AssertionError"}, "steps": [{"name": "计算两个数相乘积", "status": "passed", "start": 1613916340410, "stop": 1613916340410}], "attachments": [{"name": "stdout", "source": "2d98f8aa-45aa-4ac4-a0b3-7d57ad1f4d86-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_mul_datas", "value": "[-1, -1, 2]"}], "start": 1613916340410, "stop": 1613916340410, "uuid": "1161b687-a127-4f61-9e9b-1373881b0655", "historyId": "f11c0bd982fcae1850533f7b40d33991", "testCaseId": "fe6bda14bd0e8d9a8af1eb7bd6c20851", "fullName": "test_calc_new.TestCalc#test_mul", "labels": [{"name": "feature", "value": "测试计算器"}, {"name": "story", "value": "测试乘法"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "ELF"}, {"name": "thread", "value": "6812-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}